CMakeLists.txt

```diff
@@ -186,7 +186,7 @@ if (ENABLE_QT)
         set(QT_PREFIX_HINT)
     endif()
 
-    find_package(Qt5 REQUIRED COMPONENTS Widgets Multimedia ${QT_PREFIX_HINT})
+    find_package(Qt5 REQUIRED COMPONENTS OpenGL Widgets Multimedia ${QT_PREFIX_HINT})
```

Comment:
FYI: Took me forever to figure out this bug! citra's prebuild step couldn't find the `OpenGL` library because it was missing from this `find_package` call.

I have no idea why this isn't an issue upstream, but this is a very simple and contained fix.

Reply:
Very odd, and it builds perfectly fine for me locally. Can you document this change with a comment referring back to this PR so if conflicts ever pop up during updates, we know the reasoning?

Reply:
Done. I also moved it to its own line so that we're less likely to have merge conflicts.